lib/commands/token.js

```diff
@@ -16,7 +16,7 @@ async function paginate (href, opts, items = []) {
 class Token extends BaseCommand {
   static description = 'Manage your authentication tokens'
   static name = 'token'
-  static usage = ['list', 'revoke <id|token>', 'create --name=<name> [--token-description=<desc>] [--packages=<pkg1,pkg2>] [--packages-all] [--scopes=<scope1,scope2>] [--orgs=<org1,org2>] [--packages-and-scopes-permission=<read-only|read-write|no-access>] [--orgs-permission=<read-only|read-write|no-access>] [--expires=<days>] [--cidr=<ip-range>] [--bypass-2fa] [--password=<pass>]']
+  static usage = ['list', 'revoke <id|token>', 'create [options]']
```

Comment:
```suggestion
  static usage = ['list', 'revoke <id|token>', 'create']
```

These are for positional args.  We don't have a good way to isolate options to a single command so they're just left to `options` below.